src/Microsoft.Diagnostics.Runtime/src/DacInterface/SosDac.cs

```diff
@@ -344,13 +344,25 @@ public bool GetRCWData(ulong rcw, out RCWData data)
             return SUCCEEDED(hr);
         }
 
+        public ClrDataModule? GetClrDataModule(ulong module)
+        {
+            if (module == 0)
+                return null;
+
+            InitDelegate(ref _getModule, VTable->GetModule);
+            if (_getModule(Self, module, out IntPtr iunk) != S_OK)
+                return null;
+
+            return new ClrDataModule(_library, iunk);
+        }
+
         public MetaDataImport? GetMetadataImport(ulong module)
         {
             if (module == 0)
                 return null;
 
-            InitDelegate(ref _getMetaData, VTable->GetMetaDataImport);
-            if (_getMetaData(Self, module, out IntPtr iunk) != S_OK)
+            InitDelegate(ref _getModule, VTable->GetModule);
```

Comment:
This isn't intentional right?

Reply:
From what I saw in [sospriv.idl](https://github.com/dotnet/runtime/blob/137aa8460fd18788b00d53b22ab127bbba5f9132/src/coreclr/src/inc/sospriv.idl#L197) or [sospriv.h](https://github.com/dotnet/runtime/blob/137aa8460fd18788b00d53b22ab127bbba5f9132/src/coreclr/src/pal/prebuilt/inc/sospriv.h#L747) the method name is `GetModule`.

Reply:
You have `GetClrDataModule` and `GetMetadataImport` both using the same `VTable->GetModule`, which can't be right?

Reply:
Or do both go through the same VTable slot, and the QI determines the interface?

Reply:
Ah nevermind, I get it now.  =)